### DISCUSSION TITLE
Episodio

### DISCUSSION THREAD

**Discussion by stmorales** [JavierBZ/Tarea3-IIC3697#1] [Q&A] [answered]
Hola, no me queda claro qué es un episodio en el contexto de la tarea. Entiendo que es un step y las épocas, pero un episodio es una "partida" hasta que el jugador pierde?

Gracias

**Comment by JavierBZ** [comment, marked as answer]
Si eso mismo, un episodio dura desde que partes jugando (con env.reset() ) hasta que se termina cuando el ambiente devuelve "done=True"

**Reply by cacoos** [reply]
Hola Javier!

No sé si estoy mal, pero me da que por época hay un total de 75 episodios aprox.

Lo encuentro raro, porque serían como 3000 iteraciones por cada episodio 🤔

Está bien? si lo está, por qué calcular el promedio de los últimos 100 episodios si solo tengo 75?

Gracias!!

**Reply by JavierBZ** [reply]
Hola caco!

Si revisan documentación asociada a la implementación en gym de Enduro pueden encontrar que el max número de pasos por episodio puede llegar hasta 10.000, asi es que es posible que hayan menos de 100 episodios por época.

Por lo que dices que te dieron 75, lo probe haciendo acciones aleatorias y me dio lo mismo. No lo he encontrado en documentación, pero mi pronostico es que el juego termina al primer día, si es que no ha sobrepasado un cierto número de autos. Y el término de ese día debe estar cerca de 1/3 *10.000 pasos. 

Espero te sirva!

**Reply by cacoos** [reply]
Ahhh perfecto, grande Javier, muchas gracias, te pasaste!

**Reply by JavierBZ** [reply]
Me faltó decirte sobre sólo calcular 75, la idea de ese calculo es tener un gráfico de promedio móvil. En este juego es posible que ese promedio tome datos de más de una época y no hay problema con eso. Es sólo una manera estándar de medir rendimientos en problemas de aprendizaje reforzado

**Reply by cacoos** [reply]
Perfect 👌
